__tests__/e2e/utils.ts

```diff
@@ -62,10 +69,11 @@ export interface MockedFetch extends Fetch {
 }
 
 jest.mock('node-fetch', () => {
+  const { JSDELIVR_BASE, UNPKG_BASE, JSDELIVR_API_BASE } = require('../../src/consts');
```

Comment:
maybe the mock is better in a separate file instead of in `utils`? dunno 🤷‍♂️

Reply:
Did better, there is a test CDN implementation which is removing the need to do any fetch mock